contracts/p0/assets/Collateral.sol

```diff
@@ -63,8 +60,7 @@ contract CollateralP0 is ICollateral, AssetP0, Context {
 
         // If the underlying fiatcoin price is below the default-threshold price, default eventually
         if (whenDefault > block.timestamp) {
-            Price memory p = fiatcoinPrice(); // {attoPrice/fiatTok}
-            whenDefault = p.quantity(uoa).lte(_defaultThreshold())
+            whenDefault = fiatcoinPrice().lte(main.defaultThreshold())
```

Comment:
🤔  -- bad smell here; we'd previously been treating main.defaultThreshold as a ratio, not a price. Maybe we need `function minFiatcoinPrice()`, which for our current assets is just `return main.defaultThreshold();`?

Reply:
Good catch here, was going to fast and didn't catch the units. Applying your suggested fix